src/software/thunderscope/thunderscope_main.py

```diff
@@ -38,7 +38,7 @@
 
 if __name__ == "__main__":
 
-    logging.getLogger().setLevel(logging.INFO)
+    logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
do we want this change?

Reply:
It's handy for debugging :), will remove before final reviews and merge